### PR TITLE
Update apps.toml for packages not maintained

### PR DESCRIPTION
**Pull request by MadMan247**
Looking for feedback on my current selection. I am not quite certain what the official yunohost maintainer definition of "package not maintained" is, so please correct me if I am mistaken, and please feel free to suggest edits.

I am generally looking for packages unmaintained in 3 months, with a more recent upstream update existing, or nonfunctioning due to a lack of maintainence. If an upstream has recent commits, but no recent releases since the yunohost package was last updated, then the ynh package is in good standing by my metrics.

Developed but unusable (recent development, but broken)
Discourse
Lychee

Questionable:
Grafana? (minimally maintained, not up to date, but last commit only 1 mo ago)
bicbucstriim (updated 1 mo ago to a version that's 11 mo old) 1.6.5. vs 3.3.0
calibre-web - very strange versioning on yunohost. version 96.21 vs 6.21? typo?
chitchatter -- looks very underdeveloped on ynh, 6 mo since last ynh push w/ active upstream development, no releases. 
chtickynotes - no upstream? - remove upstream line if there is no upstream in manifest.toml.
Borg - no upstream? Upstream exists, https://github.com/borgbackup/borg

Abantecart: no upstream updates in 10 mo, no ynh updates in 7 mo
Adminer:  no upstream updates in 3 years, no ynh updates in 7 mo
AgenDav: no upstream 2 years, no ynh 5 mo
beehive: no upstream 3 years, no ynh 3 mo
biboumi: no upstream 2 years, no ynh 5 mo
blogotext: no upstream 5 years, no ynh 3 mo

I went through app list up to Cinny. Will continue when I have more free time.

### CODE REVIEW

File: apps.toml [hunk 1]
```diff
@@ -142,6 +142,7 @@ url = "https://github.com/YunoHost-Apps/anarchism_ynh"
 
 [archivebox]
 added_date = 1674232499 # 2023/01/20
+antifeatures = [ "package-not-maintained" ]
```
Comment (alexAubin) [inline apps.toml:145]: I'm a bit confused, both the upstream and the package seem somewhat active / maintained ?

Comment (MadMan247) [reply apps.toml:145]: If you disagree, I can accept that entirely, but allow me to build my case:
Upstream is absolutely active and maintained, but I believe that the yunohost package is undermaintained.
The yunohost package has not received attention for 3 months, and I believe that the last time it was updated to match version 0.7.2, it was done improperly. [Please see my pull request](https://github.com/YunoHost-Apps/archivebox_ynh/pull/37) [and my issue](https://github.com/YunoHost-Apps/archivebox_ynh/issues/36).
I also believe that it otherwise needs attention to polish missing or nonfunctioning pull methods.

a version of archivebox installed on my ubuntu (non-yunohost) machine using the bash setup script (without docker)
Screenshot from 2024-05-29 21-22-30
Please note that the internet archive rarely works (works 10% of the time)

A version of archivebox using this yunohost installation
Screenshot from 2024-05-29 21-26-13
Please note the 0.6.2 watermark, which is a version that is 4 years old. Singlefile is nonfunctional.

**Comment by Salamandar** [comment]
TBH I'm not saying your PR is bad, but we need an automated way to set this value. If tomorrow, an unmaintained package becomes maintained again, it will most probably stay flagged as unmaintained here…

**Comment by MadMan247** [comment]
I understand. However, I would imagine that, as a part of maintaining a package, which is already an effortful task, a small amount more effort could be put to submit a pull request here to remove the package not maintained tag. If the argument is that there is a delay from them maintaining it to the pull request being approved, the maintainer is intended to be a long term role, so I believe that would be appropriate?

Further, I would argue that it is easier to detect maintainence than it is to detect non-maintainence (there's a better word. Neglect?). Someone makes a few commits? Maintainence. Someone doesn't make a commit for a month? Did they drop it, or is it just good as-is? 3 months? 6 months? Where's that line?

My final argument is one of an end user: would it not be better to warn a user of potential problems where there are none, than not warn a user where there are some? (+ maybe someone will see it and step in to help maintain it)

My experience with yunohost is limited, so I do appreciate your insight. As combing through all of the services is time consuming, I am willing to stop, or change my criteria so that more severe cases are only listed (e.g. no ynh updates in 8 mo, package is a 3 yr old ver)

For automation, I can see a few methods to determine this: 
1. Maintainer in manifest.toml. If blank...
2. Would have to watch upstream for commits, but mostly releases.
3. Check if issues are closed/labled
4. Check if pull requests are handled
5. Ignore commits by yunohost-bot etc
If you're going this automation route, I could see a possible option to email the maintainer periodically to confirm maintainer status. If they take a year off it could remove them as a maintainer automatically, so someone else could pick it up in the meantime.

**Comment by ericgaspar** [comment]
I will say it's a good issue but a bad PR. 😬

**Comment by MadMan247** [comment]
> I will say it's a good issue but a bad PR. 😬

I appreciate the feedback, but what can I do to improve it?

**Comment by ericgaspar** [comment]
after a quick search:
```
Automad -> latest upstream version
Bludit -> latest upstream version
chitchatter -> new version on testing branch, waiting to be tested and merged
Cockpit -> relies on Debian distributed package, will be updated with Bookworm
Converse -> 1 version behind
Coturn -> relies on Debian distributed package, will be updated with Bookworm
Cryptpad -> new version on testing branch, failing tests, waiting for a debug
Deluge -> few version behind (last commit 4 months ago) 
Etherpad_mypads -> new version on testing branch, failing tests, waiting for a debug
Facilmap -> few version behind (last commit 2 months ago) 
Farum -> few version behind (last commit 3 months ago) 
Lychee -> latest upstream version
Piwigo -> new version on testing branch, failing tests, waiting for a debug
...
```
Taging these repositories as unmaintained feels like a kick in the butt  to the maintainers who still aren't doing enough to keep the package up to date.

**Comment by MadMan247** [comment]
Thank you, Ericgaspar, for your time. I certainly failed at least for Bludit, and automad. Thank you for taking the time to review this commit, because I aspire to do a good job.

For automad, I neglectfully failed to notice that the 2.0.0 was an alpha release.
I'm still confused about chitchatter, but I'm going to hold here, since the ynh testing branch's only commit ahead of master is you stepping down as maintainer.
Deluge- Last commit was you removing yourself as a maintainer, and should not be considered an actual commit for the sake of maintenance. Unless the 5-year-old version is intentional, I see this as unmaintained.
For cockpit or coturn, is there a way for me to know this in the future? Should there be a warning that it has to be such an outdated version?
Converse: more than one version behind-- ynh on 10.1.4, released since are 1.5, 1.6, 1.7. | 10.1.5 released almost a year ago, and ynh is still not updated to it.
Cryptpad/Etherpads MyPads: I apologize, I failed to check the testing branch. Thank you for your efforts!
Facilimaps: While I know that you said that it wasn't unmaintained, you went and maintained it to a version that isn't 3 years old, and that's a mere part of why you are a essential pillar of this operation. Thank you!
For Flarum, I am seeing the [latest upstream version as 1.8.5](https://github.com/flarum/framework/releases/tag/v1.8.5)? ynh is 1.8.2.
For Lychee and Piwigo, in all fairness I likely did categorize them incorrectly due to them having critical unresolved bugs preventing their use. Not sure if the issues are resolved, but the corresponding issues are still open on their githubs.

My request for review from alex was a mistake. Did not know that I did that. Apologies for the trouble.

I do apologize, as I am not sure that I understand your last sentence - it feels like a kick in the butt for the maintainers who are not doing enough to keep the package up to date? While I would not want to insult or disparage someone who is working on maintaining, isn't the point of the tag to indicate a package that is not being properly maintained? If the package is not being properly maintained and up to date with the upstream, that's bad for yunohost as a whole. 

TL/DR: My bad on Automad, Bludit, Cryptpad, and Ether_Mypads. + Thank you!!! Lychee and Piwigo mislabled as unmaintained because they have critical bugs preventing proper use.
